changelog.md

```diff
@@ -1,19 +1,23 @@
 # Changelog
 
-## v0.6.0 | t.b.d.
+## v0.6.0 | 2020-08-18
+
+* Plot and return images and timestamps for scans using physical coordinate system rather than fast and slow scanning axis. See docs tutorial section on `Confocal images` and `Kymographs` on how to use the API. **Note that this is a breaking change for scans with the fast axis in y direction!**
```

Comment:
What's the nature of the breaking change? Are users going to suddenly get wrong results or were their old results wrong? What do they do?

Reply:
Ah, okay. Yeah, I see the lack of explanation now, thanks. I've tried to make it more explicit.